### PR TITLE
Add advisories for 2.2.0 core-kit third party packages

### PR DESCRIPTION
**Pull request by vigh-m**
**Description of changes:**

Add Bottlerocket Security Advisories for the `2.2.0` release of the bottlerocket-core-kit. 

This PR is just for the third-party package CVEs fixed in 2.2.0. Kernel advisories in #37

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by yeazelm — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

File: advisories/2.2.0/BRSA-wyuvthdr.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[advisory]
+id = "BRSA-wyuvthdr"
+title = "soci-snapshotter CVE-2024-24788"
```
Comment (bcressey) [inline advisories/2.2.0/BRSA-wyuvthdr.toml:3]: This is a Go-related vulnerability. Did it ever affect the soci-snapshotter package shipped in the core kit?

Comment (bcressey) [reply advisories/2.2.0/BRSA-wyuvthdr.toml:3]: > This is a Go-related vulnerability. Did it ever affect the soci-snapshotter package shipped in the core kit?

Oh, I see - it also required a `golang.org/x/net` update on the SOCI side. So this is fine.

**Review by bcressey — APPROVED** [review]

File: advisories/2.2.0/BRSA-tjnwgl2a.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[advisory]
+id = "BRSA-tjnwgl2a"
+title = "docker CVE-2024-29018"
+cve = "CVE-2024-29018"
+severity = "high"
+description = "A flaw in the `dockerd` design allowed for a potential data exfiltration from 'internal' networks via authoritative DNS servers. This is because `dockerd` will forward DNS requests to the host loopback device, bypassing the container network namespace's normal routing semantics entirely."
+
+[[advisory.products]]
+package-name = "docker"
```
Comment (bcressey) [inline advisories/2.2.0/BRSA-tjnwgl2a.toml]: Need to align these package names with the package name from the spec file:
```suggestion
package-name = "docker-engine"
```

**Review by bcressey — DISMISSED** [review]

**Comment by vigh-m** [comment]
Update `package-name` as suggested

**Review by ginglis13 — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Review by KCSesh — APPROVED** [review]

**Pull request merged**